groups/sig-testing/groups.yaml

```diff
@@ -169,6 +169,32 @@ groups:
   # required for membership in these groups.
   #
 
+  - email-id: k8s-ci-robot@kubernetes.io
+    name: k8s-ci-robot
+    description: |-
+      Github admins for github account k8s-ci-robot
+    settings:
+      ReconcileMembers: "true"
+      WhoCanPostMessage: "ANYONE_CAN_POST"
+      WhoCanViewMembership: "ALL_MEMBERS_CAN_VIEW"
+    members:
+    - sig-k8s-infra-leads@kubernetes.io
+    - github@kubernetes.io
+    - sig-testing-leads@kubernetes.io
+
+  - email-id: k8s-github-robot@kubernetes.io
+    name: k8s-github-robot
+    description: |-
+      Github admins for github account k8s-github-robot
+    settings:
+      ReconcileMembers: "true"
+      WhoCanPostMessage: "ANYONE_CAN_POST"
+      WhoCanViewMembership: "ALL_MEMBERS_CAN_VIEW"
+    members:
```

Comment:
can we promote someone to OWNER of this?
right now nobody can e.g. moderate, as we only have members